dotcom-rendering/src/components/MainMediaGallery.tsx

```diff
@@ -36,16 +46,40 @@ export const MainMediaGallery = ({ mainMedia, format }: Props) => {
 
 	return (
 		<figure css={styles}>
-			<Picture
-				alt={mainMedia.data.alt ?? ''}
-				format={format}
-				role={mainMedia.role}
-				master={asset.url}
-				width={width}
-				height={height}
-				loading="eager"
-				isMainMedia={true}
-			/>
+			{renderingTarget === 'Apps' ? (
+				<Island priority="critical">
+					<AppsLightboxImage
+						elementId={mainMedia.elementId}
+						role={mainMedia.role}
+						format={format}
+						master={asset.url}
+						alt={mainMedia.data.alt ?? ''}
+						width={width}
+						height={height}
+						loading={'lazy'}
```

Comment:
I think you want "eager"[^1] for main media? Lazy loading is a performance optimisation to defer loading of images until they appear within the viewport, but a main media on galleries is the first feature you see on the page, so it will _always_ appear within the viewport on initial load.

```suggestion
						loading="eager"
```

[^1]: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading